src/diffusers/models/unet_2d_blocks.py

```diff
@@ -503,6 +503,29 @@ def forward(self, x):
 
 
 class UNetMidBlock2D(nn.Module):
+    """
+    A 2D UNet mid-block [`UNetMidBlock2D`] with multiple residual blocks and optional attention blocks.
+
+    Args:
+        in_channels (`int`): The number of input channels.
+        temb_channels (`int`): The number of temporal embedding channels.
+        dropout (`float`, *optional*, defaults to 0.0): The dropout rate.
+        num_layers (`int`, *optional*, defaults to 1): The number of residual blocks.
+        resnet_eps (`float`, *optional*, 1e-6 ): The epsilon value for the resnet blocks.
+        resnet_time_scale_shift (`str`, *optional*, defaults to `default`): The time scale shift for the resnet blocks.
+        resnet_act_fn (`str`, *optional*, defaults to `swish`): The activation function for the resnet blocks.
+        resnet_groups (`int`, *optional*, defaults to 32): The number of groups for the resnet blocks.
```

Comment:
Could we maybe add some context on how this parameter is applied in the block? 

e.g. 
resnet_groups: The number of groups to use in the group normalization layers of the resnet.
resnet_timescale_shift: The type of normalization to apply to the time embeddings.

Reply:
Okay, I understand, Now I added a new one that looks like that if it's OK.

* `resnet_groups`: The number of groups to use in the group normalization layers of the resnet blocks. This can help to improve performance and reduce memory usage.

* `resnet_time_scale_shift`: The type of normalization to apply to the time embeddings. This can help to improve the performance of the model on tasks with long-range temporal dependencies.

Reply:
Hi @hi-sushanta. I think the phrase "improve performance of the model" is dependent on a number of factors, and reducing the number of groups doesn't necessarily lead to memory savings. So I would remove those additional sentences.

Reply:
Okay, I am done to remove additional sentences.